pkg/actors/placement/placement_interface.go

```diff
@@ -11,7 +11,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package internal
```

Comment:
Please create a new package for the interfaces, don't move them inside `placement` which is a concrete implementation.

Reply:
No problem. Will move.